appsdk/sdk.go

```diff
@@ -279,10 +359,31 @@ func (sdk *AppFunctionsSDK) listenForConfigChanges() {
 				return
 			}
 
-			sdk.config.Writable = *actual
+			previousLogLevel := sdk.config.Writable.LogLevel
 
-			sdk.LoggingClient.Info("Writeable configuration has been updated from Registry")
+			sdk.config.Writable = *actual
 			sdk.LoggingClient.SetLogLevel(sdk.config.Writable.LogLevel)
+			sdk.LoggingClient.Info("Writable configuration has been updated from Registry")
+
+			if previousLogLevel != sdk.config.Writable.LogLevel {
+				// Log level changed, not Pipeline, so skip updating the pipeline
+				continue
+			}
+
+			if sdk.usingConfigurablePipeline {
```

Comment:
Should we log a message if someone is modifying the Pipeline when the configurable isn't being used?